configuration/security.md

```diff
@@ -218,20 +296,33 @@ Limit login attempts on ssh port using tcp: this denies connection if an IP addr
 sudo ufw limit ssh/tcp
 ```
 
-Deny access to port 30 from IP address 192.168.2.1
+Deny incoming requests from IP address 192.168.2.1 to port 22 only
 
 ```bash
-sudo ufw deny from 192.168.2.1 port 30
+sudo ufw deny in proto tcp from 192.168.2.1 to any port 22
 ```
 
+Allow incoming requests from IP address 192.168.2.1 to port 22 only
```

Comment:
What's the difference between "Allow incoming requests from ..." and "Limit incoming requests from ..." ?

(sorry for being picky, but I think if we're advising (potentially) novice users on how to "harden" their system, we need to be clear exactly what we're doing in order that they don't accidentally enable / disable something they didn't mean to)

Reply:
The difference is that by limiting incoming requests you're adding brute-force protection by only allowing 6 requests to be made from that IP address every 30 seconds. I can add some explanation for this.

Reply:
Everything needs to be thoroughly explained - this is a beginners guide so everything has to be crystal clear in its explanations and its implications.

Reply:
Fair enough, I've just added a fair bit of explanation and I'll be looking through it to see if I should add anything else.

Reply:
Extra explanatory text looks good :+1: